algebird-core/src/main/scala/com/twitter/algebird/ScalableBloomFilter.scala

```diff
@@ -0,0 +1,94 @@
+package com.twitter.algebird
+
+import com.googlecode.javaewah.{ EWAHCompressedBitmap => CBitSet }
+import scala.collection.mutable.{ BitSet => MBitSet }
+
+object ScalableBloomFilter {
+
+  /**
+   * The default parameter values for this implementation are based on the findings in "Scalable Bloom Filters",
+   * Almeida, Baquero, et al.: http://gsd.di.uminho.pt/members/cbm/ps/dbloom.pdf
+   */
+  def apply(fpProb: Double,
+    initialCapacity: Int,
+    growthRate: Int = 2,
+    tighteningRatio: Double = 0.9) = {
+    new ScalableBloomFilter(fpProb, initialCapacity, growthRate, tighteningRatio,
+      List(BloomFilter(initialCapacity, fpProb).create()))
```

Comment:
fpProb is 'P' from the paper, while we need to specify P0 here.  By my reading we should be using:

BloomFilter(initialCapacity, fpProb * (1 - tighteningRatio))